_pages/code-of-conduct.md

```diff
@@ -96,27 +96,27 @@ Admins and other leaders who do not follow or enforce the Code of Conduct in goo
 
 The vast majority of situations work out like this, in our experience. This interaction is common, and generally positive.
 
-> Alex: "Yeah I used X and it was really crazy!"
+> Alex: "Yeah I used X and it was [negative association with mental health]"
```

Comment:
Nice. Good point that including the language itself can server to unnecessarily re-traumatize.